### PR TITLE
[Merge] Release 0.5.0

### PR DESCRIPTION
**Pull request by SferaDev**

### CODE REVIEW

File: src/webapp/components/module-list-table/ModuleListTable.tsx [hunk 1]
```diff
@@ -181,7 +226,8 @@ export const ModuleListTable: React.FC = () => {
                 onClick: deleteModules,
                 isActive: _rows => {
                     // TODO: Action is disabled for now
-                    return false;
+                    //return false;
```
Comment (SferaDev) [inline src/webapp/components/module-list-table/ModuleListTable.tsx]:
```suggestion
```

File: src/webapp/components/module-list-table/ModuleListTable.tsx [hunk 2]
```diff
@@ -53,8 +56,9 @@ export const ModuleListTable: React.FC = () => {
             setRefreshKey(Math.random());
             setTableLoading(false);
             setSelection([]);
+            snackbar.info("Logic not implemented yet");
```
Comment (SferaDev) [inline src/webapp/components/module-list-table/ModuleListTable.tsx]:
```suggestion
```

File: src/webapp/components/module-list-table/ModuleListTable.tsx [hunk 3]
```diff
@@ -53,8 +56,9 @@ export const ModuleListTable: React.FC = () => {
...
         },
-        [usecases]
+        [usecases, snackbar]
```
Comment (SferaDev) [inline src/webapp/components/module-list-table/ModuleListTable.tsx]:
```suggestion
        [usecases]
```

**Review by ifoche — APPROVED** [review]
Thanks @SferaDev all issues seems to be cleared. Just for the record, everything's in the following clickup card:
https://app.clickup.com/t/dk4yya

**Pull request merged**